packages/node_modules/node-red/settings.js

```diff
@@ -162,22 +97,75 @@ module.exports = {
     //     }
     // },
 
-    // The following property can be used to refresh the https settings at a
-    // regular time interval in hours.
-    // This requires:
-    //   - the `https` setting to be a function that can be called to get
-    //     the refreshed settings.
-    //   - Node.js 11 or later.
+    // If the `https` setting is a function, the following setting can be used
+    // to set how often, in hours, the function will be called. That can be used
+    // to refresh any certificates.
     //httpsRefreshInterval : 12,
 
     // The following property can be used to cause insecure HTTP connections to
     // be redirected to HTTPS.
     //requireHttps: true,
 
-    // The following property can be used to disable the editor. The admin API
-    // is not affected by this option. To disable both the editor and the admin
-    // API, use either the httpRoot or httpAdminRoot properties
-    //disableEditor: false,
+    // To password protect the node-defined HTTP endpoints (httpNodeRoot), or
```

Comment:
Should we mention this is also used to protect the node-red-dashboard ?

Reply:
done